scripts/history-update-check.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+#
+# Check each PR has an entry in HISTORY.md during a CI routine.
+#
+# Arguments:
+#   $ history-update-check.sh
+#
+# This should be run from the root of the repository.
+set -e
+
+! git diff --exit-code master -- HISTORY.md > /dev/null
+
+if [ $? ]; then
+    echo "History log was updated ..."
+else
+    echo "History log is missing an entry for this PR ..."
+    echo "Please update the history log to run CI ..."
```

Comment:
I'd change to a slightly more descriptive return message such as "Please describe your changes in HISTORY.md" Otherwise, this is great! I love how simple it is.

Reply:
@birm can you make the phrase a suggestion using:

````
```suggestion
    echo "Please update the history log to run CI ..."
```
````

Reply:
Oh, that's how you do that! In any case, it looks like you did it.